prowler/providers/aws/services/neptune/neptune_cluster_backup_enabled/neptune_cluster_backup_enabled.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.neptune.neptune_client import neptune_client
+
+
+class neptune_cluster_backup_enabled(Check):
+    def execute(self):
+        findings = []
+        for cluster in neptune_client.clusters.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = cluster.region
+            report.resource_id = cluster.name
+            report.resource_arn = cluster.arn
+            report.resource_tags = cluster.tags
+            report.status = "FAIL"
+            report.status_extended = (
+                f"Neptune Cluster {cluster.name} does not have backup enabled."
+            )
+            if (
+                cluster.backup_retention_period > 0
+                and cluster.backup_retention_period < 7
```

Comment:
Could you do this as a configurable variable, please?

Reply:
Do you have an example to reference? I want to make sure Im following the code style so it does not cause any compatibility issues. Will also need to add some additional tests cases for this as well.

Reply:
Sure! You can find it here: https://github.com/prowler-cloud/prowler/blob/9ca64e7bdb46d4f5791140a37946ca8ae98d4ccf/prowler/providers/aws/services/acm/acm_certificates_expiration_check/acm_certificates_expiration_check.py#L12

Reply:
Thank you for the example. Added that into the check and an additional test case as well to verify it was being successfully set.